### DISCUSSION TITLE
Every table a printer

### DISCUSSION THREAD

**Discussion by jhonnyjoao** [OrcaSlicer/OrcaSlicer#1037] [Ideas]
I think it would be interesting to be able to change the printer for each table! It would be very useful for those who have more than one printer. Each one with its settings and its material
![Screenshot 2023-05-11 085823](https://github.com/SoftFever/OrcaSlicer/assets/89656487/d3ec5466-4872-4013-bd58-f69bd580102e)

**Comment by gasprints** [comment]
I second this
